CHANGELOG.md

```diff
@@ -25,19 +25,24 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/) and this p
 - server_security_misconfiguration.clickjacking.form_input
 - server_security_misconfiguration.misconfigured_dns.basic_subdomain_takeover
 - server_security_misconfiguration.misconfigured_dns.high_impact_subdomain_takeover
+- server_security_misconfiguration.captcha
```

Comment:
maybe this changelog entry should be under 'changed'?

i.e. `server_security_misconfiguration.captcha_bypass` changed to `server_security_misconfiguration.captcha`

Reply:
That's an alternative, but a new type of entry at the same time. @barnett any thoughts?

Reply:
Looking at the history of the changelog this actually fits better.  'changed' is essentially for non id changes.  Any change to id is in added/removed which makes sense 👍